### DISCUSSION TITLE
Delete entity workflow

### DISCUSSION THREAD

**Discussion by diderich** [YetiForceCompany/YetiForceCRM#16406] [How to]
Is there a possibility to create a workflow that deletes records that satisfy specific conditions?

**Comment by rskrzypczak** [comment]
By default, this is not possible.
You would have to create your own workflow task.

**Comment by s2b-git** [comment]
This would be helpful as a default task in light of limited control of duplicates. Possibly more reasons could be found. 
[diderich](https://github.com/diderich) did you managed to fix it?

Best
